### ISSUE TITLE
pVACview: neofox_ui.R and custom_ui.R not found in the output folder

### ISSUE THREAD

**Issue by LukaszGraff** [griffithlab/pVACtools#1128]
### Installation Type

Docker

### pVACtools Version / Docker Image

griffithlab/pvactools:latest

### Describe the bug

neofox_ui.R and custom_ui.R are not found in the output folder, but app.R has references to them showing errors due to their absence and failing to run.

### How to reproduce this bug

```Shell
Run any command to generate the output files. Then run app.R.
```

### Log output

Warning in file(filename, "r", encoding = encoding) :
  cannot open file 'neofox_ui.R': No such file or directory
Warning: Error in file: cannot open the connection

### Output files

[MHC_Class_I.zip](https://github.com/user-attachments/files/16442115/MHC_Class_I.zip)

**Comment by susannasiebert** [comment]
Thank you for bringing this to our attention. I have made a bugfix PR (#1126) to address this issue, which should go out with the next hotfix release. In the meantime, as a workaround, you can manually download/copy the missing R files from [GitHub](https://github.com/griffithlab/pVACtools/tree/master/pvactools/tools/pvacview).

**Comment by susannasiebert** [comment]
This issue has been fixed in version 4.3.0. Please give it a try and reopen this issue if you're still encountering this error.
